neuralop/models/uno.py

```diff
@@ -153,19 +153,25 @@ def __init__(self,
         self.separable = separable
         self.preactivation = preactivation
         self._incremental_n_modes = incremental_n_modes
-        self.output_scaling_factor = output_scaling_factor
         self.operator_block = operator_block
         self.integral_operator = integral_operator
         
         if self.horizontal_skips_map is None:
             self.horizontal_skips_map = {}
             for i in range(n_layers//2,0,):
                 self.horizontal_skips_map[n_layers - i -1] = i
-                
+
+        self.output_scaling_factor = np.ones_like(self.uno_scalings[0])
+        for k in self.uno_scalings:
+            self.output_scaling_factor = np.multiply(self.output_scaling_factor, k)
+        self.output_scaling_factor = self.output_scaling_factor.tolist()
```

Comment:
Let's make it a list and not add additional `tolist`.
You can replace the np.multiply (which should work on a list anyway) with math.prod.
Actually we don't need the list, you can directly take the prod I think

Reply:
math.prod multiply numbers in a list. but I need to multiply 2 lists. Each list specifies the scaling factors along each of the dims.

Reply:
Hmm this code is becoming very confusing - can we try to think and simplify this scaling API?

Reply:
To do the scaling we need two things ---
one -- scaling factor for each layer.
two -- scaling factor along each of the dimension.
We need a two array in general.
But in case, we scale each of the dimensions with same scaling factor, then it can be a 1d array, where each element is the scaling factor for each layer.

Reply:
Alright, let's process it to always be a nested list, at the highest level and we can use zip to do the element-wise multiplication. 

Do we **need** to have both output_scaling_factor and uno_scaling?

Reply:
?